TombEngine/Renderer/Native/DirectX11/DX11SpriteFont.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
```

Comment:
By the way, are we adopting OpenLara way of doing things, and prefer to keep all code in .h files now?